pulsar/common/attackchains/topics.go

```diff
@@ -2,12 +2,9 @@ package attackchains
 
 const (
 	// topic names
-	AttackChainStateScanStateTopic   = "attack-chain-scan-state-v1"
-	AttackChainStateViewedTopic      = "attack-chain-viewed-v1"
-	AttackChainStateDeleteTopic      = "attack-chain-delete-v1"
-	KubescapeScanReportFinishedTopic = "kubescape-scan-report-finished-v1"
-	ContainerScanReportFinishedTopic = "container-scan-report-finished-v1"
-	AttackChainScanReportFinishTopic = "attack-chain-scan-report-finished-v1"
+	AttackChainStateScanStateTopic = "attack-chain-scan-state-v1"
```

Comment:
Why are we leaving it here? I mean, we can move all topics under common/consts.go

Reply:
or maybe call it common/topics.go